### PR TITLE
feat: integrate related prompts

### PR DESCRIPTION
**Pull request by victorcg88**
# Pull request template
 
Integrate the RelatedPrompts into the archetype.

Missing in this PR: Tagging for RelatedPrompts.

NOTE: As the empathy instance is not generating RP's we are using a store from MotiveMarket for testing purposes.

## Motivation and context

- [ ] Dependencies. If any, specify:
- [ ] Open issue. If applicable, link:

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Change requires a documentation update

## What is the destination branch of this PR?

- [x] `Main`
- [ ] Other. Specify:

## How has this been tested?

Tests performed according to [testing guidelines](https://github.com/empathyco/x/blob/main/.github/contributing/tests.md): 

## Checklist:

- [x] My code follows the **[style guidelines](https://github.com/empathyco/x/blob/main/.github/CONTRIBUTING.md#style-guides)** of this project.
- [x] I have performed a **self-review** on my own code.
- [x] I have **commented** my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate **no new warnings**.
- [x] I have added **tests** that prove my fix is effective or that my feature works.
- [x] New and existing **unit tests pass locally** with my changes.

### CODE REVIEW

**Review by lauramargar — DISMISSED** [review]
Great job!!

File: src/components/related-prompts/related-prompts-tag-list.vue [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    -webkit-background-clip: padding-box, text;
+    background-clip: padding-box, text;
+    background-repeat: no-repeat;
+  }
+
+  .x-typewritter-animation {
+    animation: t calc(var(--n) * 0.05s) steps(var(--n)) forwards;
+  }
+
+  @keyframes t {
```
Comment (diegopf) [inline src/components/related-prompts/related-prompts-tag-list.vue:195]: I'd like to have a more readable name for the animation 😅

File: src/components/related-prompts/related-prompts-tag-list.vue [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+    background: linear-gradient(-90deg, transparent 5px, #0000 0) 10px 0,
+      linear-gradient(#575757 0 0) 0 0;
+    background-size: 0 200%;
+    -webkit-background-clip: padding-box, text;
+    background-clip: padding-box, text;
+    background-repeat: no-repeat;
+  }
+
+  .x-typewritter-animation {
+    animation: t calc(var(--n) * 0.05s) steps(var(--n)) forwards;
```
Comment (diegopf) [inline src/components/related-prompts/related-prompts-tag-list.vue:192]: I'd like to have a more readable name for the variable --n

File: src/components/search/results/results.vue [hunk 3]
```diff
@@ -56,38 +58,63 @@
...
 
+      const { relatedPrompts, selectedPrompt, selectedQuery } = useState('relatedPrompts', [
+        'relatedPrompts',
+        'selectedPrompt',
+        'selectedQuery'
+      ]);
+
+      const columns = computed(() => (isMobile.value ? 2 : 4));
+
+      const isLowResult = computed(() => 0 < x.totalResults && x.totalResults < 50);
```
Comment (diegopf) [inline src/components/search/results/results.vue:99]: The lowresults threshold is a config for the semanticQueries module. We could just pick it up from there with ```const { config } = useState('semanticQueries', ['config']);```

**Review by diegopf — CHANGES_REQUESTED** [review]

**Review by diegopf — APPROVED** [review]

**Pull request merged**